### DISCUSSION TITLE
Straight down waypoint (plane)

### DISCUSSION THREAD

**Discussion by DangerD1024** [iNavFlight/inav#8003] [Q&A]
I need to make waypoint so plane should go straight down to needed height, than continue to fly to next waypoint, is that possible?
(If waypoints positions should be different i can slightly change decimals)

**Comment by MrD-RC** [comment]
Planes can’t go straight down, they need to fly forwards. You could possibly set a loiter point with the altitude that you want to get to?

**Reply by DangerD1024** [reply]
> Planes can’t go straight down, they need to fly forwards. You could possibly set a loiter point with the altitude that you want to get to?
why can't, when you fly manually, it can =)

**Reply by MrD-RC** [reply]
Sure, I can dive vertically too. But I wouldn’t want a flight controller that has no real knowledge of where the ground is doing it. 

Loiter down is the best bet to lose altitude in place on fixed wing.

**Reply by breadoven** [reply]
https://github.com/iNavFlight/inav/pull/7644 will do what you want ... but not diving vertically, just loitering/spiralling down to the set altitude.
